queries.sql

```diff
@@ -8,3 +8,55 @@ SELECT name,escape_attempts FROM animals WHERE  weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals  WHERE weight_kg >=10.4 AND  weight_kg <=17.3;
+
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name ILIKE '%mon%';
+
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL ;
+
+COMMIT;
+
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] Kindly add the transaction for this transaction in the pull request for this milestone. This will enable us to confirm if the query was written correctly. I would advise you to select all animals from the table after the `DELETE from animals` statement. 👍 😎